### DISCUSSION TITLE
Distinction between !S and !SQ / !N and !NQ actions?

### DISCUSSION THREAD

**Discussion by ixchow** [gopro/labs#968] [General]
The [documentation](https://gopro.github.io/labs/control/actions/) indicates that "!SQ" starts "quickly" and "approximately", while "!S" starts "exactly"; while "!N" uses an exact time and "!NQ" uses an approximate time.

Discussion #240 strongly hints that the difference is that "!NQ" will power down and "!N" will not, but it would be nice to have this confirmed in the documentation more explicitly. It would also be useful to know what "approximate" behavior means in terms of the timing of the resulting action (e.g., will it always be late? always be early? no reliable behavior? always target time + camera boot time?), and if waiting until a time vs waiting for a number of seconds will have a different outcome.

**Comment by dnewman-gpsw** [comment]
Less than ten seconds, neither will shutdown.  !NQ or !SQ or !RQ etc. are all approximate, better for battery efficiency.  It will be within a few seconds for shorter sleeps, sometime early sometimes late. Long sleeps have more error, typically within tens of seconds.

**Reply by ixchow** [reply]
Thanks for the quick reply (and on a weekend, no less). That's useful to know.
